rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -138,6 +138,41 @@ public static Wallet getFederationsSpendWallet(
         return wallet;
     }
 
+    protected static Coin getAmountSentToAddresses(
+        ActivationConfig.ForBlock activations,
+        NetworkParameters networkParameters,
+        Context context,
+        BtcTransaction btcTx,
+        Address... addresses
```

Comment:
Maybe we could consider making this a list, what do you think? I found some discussions online about converting from List to array in java https://shipilev.net/blog/2016/arrays-wisdom-ancients/

I'll let you decide

Reply:
BTW, what do you think if we consider this for the latest PRs? Because this is going to create some conflicts with the other branches.

Reply:
I did a quick reading of the most relevant thoughts in the article and also the conclusion. After reading it, my opinion is we may change it to List, as we may prefer use most of the time List instead of an array(at least for me). Also making the method more straightforward and avoiding the constant conversion from List to Array and viceversa(inside the method).